recipe/recipe.yaml

```
@@ -0,0 +1,62 @@
+schema_version: 1
+
+context:
+  python_min: 3.9
```

```suggestion
  python_min: 3.8
```

You could have kept this as `3.8`, you just need to rerender the feedstock.

Can do this with

```
@conda-forge-admin, please rerender
```

---

Oh, looks like `conda-forge-admin` is "smart" enough to run commands even if it isn't tagged. Interesting.